Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -929,15 +1207,15 @@ void IsolatedTracksCone::beginJob() {
   nEVT_failL1=0;
   nTRK=0;
   
-  double tempgen_TH[22] = { 0.0,  1.0,  2.0,  3.0,  4.0,  
-			    5.0,  6.0,  7.0,  8.0,  9.0, 
-			    10.0, 12.0, 15.0, 20.0, 25.0, 
-			    30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 100};
-  for(int i=0; i<22; i++)  genPartPBins[i]  = tempgen_TH[i];
+  double tempgen_TH[nPBins_+1] = { 0.0,  1.0,  2.0,  3.0,  4.0,  
+				   5.0,  6.0,  7.0,  8.0,  9.0, 
+				   10.0, 12.0, 15.0, 20.0, 25.0, 
+				   30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 100};
```

Comment:
I think here and also above you could replace this with `std::array` and just use the nice interface for iterating trough it